rhv_checks.py

```diff
@@ -6,9 +6,11 @@
 from __future__ import division
 
 import sys
+import json
 
 from ovirtsdk4 import types
 from rhv_logconf import logger
+from utils import ssh_command
```

Comment:
You said you renamed the file, I don't see it in the PR. Am I missing something?

Reply:
Added file.